overlord/snapstate/aliasesv2.go

```diff
@@ -35,10 +35,78 @@ type AliasesStatus string
 
 // Possible global aliases statuses for a snap.
 const (
-	EnabledAliases  AliasesStatus = "enabled"
-	DisabledAliases AliasesStatus = "disabled"
+	NoYetAutoAliases        AliasesStatus = ""
+	EnabledAliases          AliasesStatus = "enabled"
+	DisabledAliases         AliasesStatus = "disabled"
+	PendingEnabledAliases   AliasesStatus = "pending:enabled"
+	PendingDisabledAliases  AliasesStatus = "pending:disabled"
```

Comment:
We discussed to indeed switch this to AliasesPending  and AutoAliasesDisabled (both bool flags)